docs/team/sp4ce-cowboy.md

```diff
@@ -3,156 +3,164 @@ layout: page
-
----
-
-### 2 – Project Contributions
-
-#### Functional Features
-
-* **Delete Transaction Command**:
-  * What it does: A CLI command to remove a transaction from UniCa$h, as specified by the user.
-  * Justification: As a finances management app, removal of transactions is an important core utility
-    especially if the user had entered the information incorrectly or if the user wants to
-    delete a transaction in general.
-  * Credits: AddressBook3's implementation of the `Delete` command
-
-* **Clear Transactions Command**:
-  * What it does: A CLI command to remove all transactions from UniCa$h.
-  * Justification: A user might want to clear all their previous transactions and
-    start over on a blank slate.
-
-
-* **Reset UniCash**: *to be added*
-* **Help Command** *to be added*
-* **Get Transactions**: *to be added*
-
-
-* **Feature Enhancements**
-  * **Find Transactions**: Added enhancements *to be added*
-
-#### User Interface Features
+for users who are proficient with CLI. It is written in Java 11, spanning about [20 KLoC](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=T16-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&isTabOnMergedGroup=true&tabOpen=true&tabType=authorship&tabAuthor=elhy1999&tabRepo=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=true&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false).
+Code contributed by me: [RepoSense](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=T16-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&tabOpen=true&tabType=authorship&tabAuthor=sp4ce-cowboy&tabRepo=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false). A summary of my contributions are given below, with more detailed
+explanations in the respective PRs.
+
+**Functional Features**
+
+* **Delete Transaction Command**: [PRs: [#62](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/62)]
+  {_**Credits:** AB3's `Delete` command._}
+  * A command to remove a specified transaction from UniCa$h.
+  **Justification:** Core utility of a finance tracking app.
+
+* **Clear Transactions Command**: [PRs:
+[#63](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/63)
+[#138](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/138/)]
+  {_**Credits:** AB3's `Clear` command._}
+  * **What it does:** A command to remove all transactions, and a parser class.
+  **Justification:** To allow the user to start on a clean slate. Parser ensures that
+  mass-deletion is intentional, adding a layer of safety.
```

Comment:
Good to make `Justifications` a new line otherwise it's one long sentence

Reply:
Rationale is simular to above comment